### PR TITLE
BOAC-2488, prototype Google calendar integration; all appointments from one calendar

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2488

Includes:
* feature flag
* new Vue component: AdvisingAppointments
* `boac/externals/google_calendar_client.py` (fault tolerant)

### CODE REVIEW

**Comment by johncrossman** [comment]
On localhost, I have it pulling from `pdg.scrum.master` calendar. My test data plus recurring standup.

image

File: boac/externals/google_calendar_client.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        # authorization flow completes for the first time.
+        with open(token_pickle_path, 'rb') as token:
+            credentials = pickle.load(token)
+    # If there are no (valid) credentials available we need user to grant access (OAuth).
+    if not credentials or not credentials.valid:
+        try:
+            if credentials and credentials.expired and credentials.refresh_token:
+                credentials.refresh(Request())
+            else:
+                client_secrets_file = app.config['GOOGLE_CLIENT_SECRETS_JSON']
```
Comment (johncrossman) [inline boac/externals/google_calendar_client.py]: Follow up PR will have `client-secrets.json` pulled down in `.ebextensions/05_download_local_configuration.config` as with other `la-deploy-configs`.

Comment (johncrossman) [reply boac/externals/google_calendar_client.py]: I've amended this PR to include simple change to `05_download_local_configuration`, as described.

**Comment by paulkerschen** [comment]
It looks like a feature! Let's make a date with it.

**Pull request merged**